melodic/distribution.yaml

```diff
@@ -772,11 +772,6 @@ repositories:
       type: git
       url: https://github.com/ros-drivers/odva_ethernetip.git
       version: indigo-devel
-    release:
-      tags:
-        release: release/melodic/{package}/{version}
-      url: https://github.com/ros-drivers-gbp/odva_ethernetip-release.git
-      version: 0.1.2-0
```

Comment:
I would suggest to just drop the `version` line but keep the rest of the release entry.